packages/babel-plugin-transform-async-to-generator/test/fixtures/async-to-generator/deeply-nested-asyncs/expected.js

```diff
@@ -11,11 +11,12 @@ let s = (() => {
       var _ref2 = babelHelpers.asyncToGenerator(function* (y, a) {
         let r = (() => {
           var _ref3 = babelHelpers.asyncToGenerator(function* (z, b) {
+            yield z;
+
             for (var _len2 = arguments.length, innerArgs = Array(_len2 > 2 ? _len2 - 2 : 0), _key2 = 2; _key2 < _len2; _key2++) {
               innerArgs[_key2 - 2] = arguments[_key2];
             }
 
-            yield z;
```

Comment:
I forget, but this change seems _more_ like what it's supposed to do, so I didn't worry about it. It's possible I looked into the reason, but I don't remember anymore.